### PR TITLE
fix: add lazurite to the build matrix

### LINKED ISSUE #195: Turn on lazurite builds
lxqt is now lazurite upstream, we should turn this on for F39:

https://gitlab.com/fedora/ostree/ci-test/

### PR DESCRIPTION
**Pull request by qoijjj**
Closes #195

### CODE REVIEW

**Review by p5 — COMMENTED** [review]
Please can you add an exclude for lazurite F38 since it looks like they're not being released.

Should look something like this on line 42
```
        exclude:
          # There is no Fedora 38 version of onyx and lazurite
          - image_name: onyx
            major_version: 38
          - image_name: lazurite
            major_version: 38
```

**Review by p5 — APPROVED** [review]
LGTM!  Thanks!

We just need another approval from a maintainer, then this can be merged!

**Review by castrojo — APPROVED** [review]

**Pull request merged**
